tests/test_geodataframe.py

```diff
@@ -30,8 +30,10 @@ def setUp(self):
         self.boros = self.df['BoroName']
         self.crs = {'init': 'epsg:4326'}
         self.df2 = GeoDataFrame([
-            {'geometry' : Point(x, y), 'value1': x + y, 'value2': x * y}
+            {'geometry': Point(x, y), 'value1': x + y, 'value2': x * y}
             for x, y in zip(range(N), range(N))], crs=self.crs)
+        self.df3 = read_file('examples/null_geom.geojson')
```

Comment:
It might be cleaner to put this in the tests directory (e.g. `tests/data`) instead. On the other hand, this is where we download the NY boroughs shapefile for tests.